app/crawler/jtbc/api.py

```diff
@@ -0,0 +1,59 @@
+from datetime import datetime
+from typing import List
+
+import aiohttp
+
+from app.crawler.base import Article, BaseNewsCrawler
+from common.utils.logger import get_logger
+
+logger = get_logger(__name__)
+
+
+class JTBCNewsApiCrawler(BaseNewsCrawler):
+    """
+    JTBC 뉴스 API 크롤러
+    """
+
+    BASE_URL = "https://news-api.jtbc.co.kr/v1/get/contents/section/list/articles"
+
+    async def fetch_articles(self) -> List[Article]:
+        logger.info("[JTBC] 뉴스 API 요청 시작")
+
+        articles: List[Article] = []
+
+        # 오늘 날짜 가져오기
+        today = datetime.now().strftime("%Y-%m-%d")
+        logger.info(f"[JTBC] 오늘 날짜: {today}")
+
+        timeout = aiohttp.ClientTimeout(total=30)
+        connector = aiohttp.TCPConnector(limit=10, ssl=False)
+
+        try:
+            async with aiohttp.ClientSession(
+                connector=connector, timeout=timeout
+            ) as session:
+                params = {"pageNo": 1, "pageSize": 20, "articleListType": "ARTICLE"}
+
+                async with session.get(self.BASE_URL, params=params) as response:
+                    if response.status == 200:
+                        data = await response.json()
+                        news_list = data["data"]["list"]
+                        for item in news_list:
+                            # 오늘 날짜의 기사만 추가
+                            pub_date = item.get("publicationDate", "").split("T")[0]
```

Comment:
datetime.strptime 사용해보실까요?

```suggestion
pub_date_ = item.get("publicationDate")
if not pub_date_:
    logger.warn("...")
    continue
pub_date = datetime.strptime(date_str, '%Y-%m-%dT%H:%M').strftime("%Y-%m-%d")
logger.info("pub_date -> %s", pub_date)
```

Reply:
오늘 날짜를 이용한 데이터 수집 로직이 삭제되면서 해당 코드도 삭제 될 예정입니다. 
다음에 작성될 로직에 날짜를 이용한 로직이 있다면 적용하도록 하겠습니다. 감사합니다^^